samples/system-test/topics.test.ts

```diff
@@ -100,6 +100,22 @@ describe('topics', () => {
     assert.strictEqual(receivedMessage.data.toString(), expectedMessage.data);
   });
 
+  it('should publish with flow control', async () => {
+    const [subscription] = await pubsub
+      .topic(topicNameThree)
+      .subscription(subscriptionNameOne)
+      .get({autoCreate: true});
+    const output = execSync(
+      `${commandFor('publishWithFlowControl')} ${topicNameThree}`
+    );
+    const receivedMessage = await _pullOneMessage(subscription);
+    assert.strictEqual(receivedMessage.data.toString(), 'test!');
+    assert.ok(output.indexOf('Published 1000 with flow control settings') >= 0);
```

Comment:
This is where chai's `assert.include` is quite nice.  It will show proper diffs when the test failed instead of a failed boolean assertion.

Reply:
I thought we were trying to work ourselves away from chai and just use the built in asserts? I'm happy to use the chai ones!